src/apiRequest.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
+const request = require('request');
+const env = require('env2')('./config.env');
+
+const buildAPIURL = ( date, apiURL = 'https://api.nytimes.com/svc/search/v2/articlesearch.json',
+  apiKey = process.env.APIKEY ) => {
+  return `${apiURL}?api-key=${apiKey}&begin_date=${date}&end_date=${date}&sort=newest`;
+}
+
+const makeRequest = ( apiUrl = buildAPIURL() ) => {
```

Comment:
can we add url as an argument and call it below with the value of the previous argument